source/loader/layers/validation/ur_validation_layer.hpp

```diff
@@ -20,19 +20,22 @@ namespace ur_validation_layer {
 ///////////////////////////////////////////////////////////////////////////////
 class __urdlllocal context_t : public proxy_layer_context_t {
   public:
-    bool enableValidation = false;
-    bool enableParameterValidation = false;
-    bool enableLeakChecking = false;
-
+    bool enableParameterValidation;
+    bool enableLeakChecking;
     logger::Logger logger;
 
     ur_dditable_t urDdiTable = {};
 
     context_t();
     ~context_t();
 
-    bool isEnabled() override { return enableValidation; };
-    ur_result_t init(ur_dditable_t *dditable) override;
+    bool isAvailable() const override { return true; }
+    std::vector<std::string> getNames() const override {
+        return {"UR_FULL_VALIDATION_LAYER", "UR_PARAMETER_VALIDATION_LAYER",
```

Comment:
Maybe these names should be exactly the same as env variables? Or we could get rid of existing env variables and replace it with this:
`UR_ENABLE_LAYERS=UR_FULL_VALIDATION_LAYER,UR_TRACING_LAYER`.

I'd also invert the names: `UR_LAYER_VALIDATION_FULL`, `UR_LAYER_TRACING`.

Reply:
yeah the `UR_LAYERS_ENABLE=list of names` approach is way nicer, it let me clean up a bunch of stuff